libs/sdk-ui-tests-e2e/scenarios/src/components/Scenarios/Filters/DateFilterScenario.tsx

```diff
@@ -1,36 +1,59 @@
-// (C) 2021 GoodData Corporation
+// (C) 2021-2022 GoodData Corporation
 
 import React, { useState } from "react";
 import { DateFilter, DateFilterOption } from "@gooddata/sdk-ui-filters";
 import { availableGranularities, defaultDateFilterOptions } from "../../constants/dateFilterConstants";
+import { props } from "lodash/fp";
 
 interface IDateFilterScenarioState {
     selectedFilterOption: DateFilterOption;
     excludeCurrentPeriod: boolean;
+    isTimeEnabled: boolean;
 }
 export const DateFilterScenario: React.FC = () => {
     const [state, setState] = useState<IDateFilterScenarioState>({
         excludeCurrentPeriod: false,
         selectedFilterOption: defaultDateFilterOptions.allTime!,
+        isTimeEnabled: false,
     });
 
     function onApply(selectedFilterOption: DateFilterOption, excludeCurrentPeriod: boolean) {
         setState({
```

Comment:
When you are spreading previous state in setState, you should always use setState callback `setState(state => ({ ...state }))`

Reply:
fixed